TS7016: Could not find a declaration file for module './fetch/errors'

**Description**

TypeScript build generates "Could not find a declaration file for module './fetch/errors'" error when trying to build a project that uses `@adobe/helix-fetch`.

**To Reproduce**
```bash
npm init -y
npm i -D typescript @types/node
npm i @adobe/helix-fetch
```

tsconfig.js:
```json
{
  "compilerOptions": {
    "target": "es2016",
    "module": "commonjs",
    "esModuleInterop": true,
    "strict": true
  }
}
```

test.ts:
```typescript
import { fetch } from '@adobe/helix-fetch';
```

Run tsc:
```bash
npx tsc
```

**Expected behavior**
No errors reported

**Actual behavior**
```
node_modules/@adobe/helix-fetch/src/api.d.ts:13:29 - error TS7016: Could not find a declaration file for module './fetch/errors'. '/test/node_modules/@adobe/helix-fetch/src/fetch/errors.js' implicitly has an 'any' type.

13 import { SystemError } from "./fetch/errors";
                               ~~~~~~~~~~~~~~~~


Found 1 error.
```

**Version:**
run: `$ hlx --version`: `Command 'hlx' not found`

`npm ls @adobe/helix-fetch`: `@adobe/helix-fetch@3.0.0`

**Additional information**

When `tsc` sees `import { SystemError } from "./fetch/errors"`, it expects to find at `./fetch/errors.ts` or `./fetch/errors.d.ts`, but neither exists. Moreover, `SystemError` does not exist in `./fetch/errors.js`:

```
/test/node_modules/@adobe/helix-fetch/src$ grep class fetch/errors.js
/* eslint-disable max-classes-per-file */
class FetchBaseError extends Error {
class FetchError extends FetchBaseError {
class AbortError extends FetchBaseError {
```

--- a/src/api.d.ts
+++ b/src/api.d.ts
@@ -10,8 +10,6 @@
  * governing permissions and limitations under the License.
  */
 
-import { SystemError } from "./fetch/errors";
-
 export declare enum ALPNProtocol {
   ALPN_HTTP2 = 'h2',
   ALPN_HTTP2C = 'h2c',
@@ -281,6 +279,19 @@ export type AbortController = Window["window"]["AbortController"];
 export interface FetchBaseError extends Error {
   type?: string;
 }
+
+interface SystemError {
+  address?: string;
+  code: string;
+  dest?: string;
+  errno: number;
+  info?: object;
+  message: string;
+  path?: string;
+  port?: number;
+  syscall: string;
+}
+
 export interface FetchError extends FetchBaseError{
   code: number;
   erroredSysCall?: SystemError;

